7-gce/config.py

```diff
@@ -106,5 +106,5 @@
 # If you receive a invalid redirect URI error review you settings to ensure
 # that the current URI is allowed.
 GOOGLE_OAUTH2_CLIENT_ID = \
-    '540860343698-of5tonk1rc7vjfgq1o6lq1vcb2oc1hsv.apps.googleusercontent.com'
-GOOGLE_OAUTH2_CLIENT_SECRET = '-OgeC9nWCSor0rEwx8B9p-hO'
+    ''
```

Comment:
https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository

Reply:
solved